server/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java

```diff
@@ -705,7 +719,7 @@ private static IndexService validateActiveShardCountAndCreateIndexService(String
 
     private void validate(CreateIndexClusterStateUpdateRequest request, ClusterState state) {
         boolean isHidden = IndexMetaData.INDEX_HIDDEN_SETTING.get(request.settings());
```

Comment:
no longer needed?

Reply:
You're right, I missed that one. Just pushed a commit removing this line. Thanks!